Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -368,6 +371,38 @@ def download_artifact(url, dir, pr)
     end
   end
 
+  def pr_check_conflicts(name, tap_remote_repo, pr)
+    pull_requests = search_issues("org:#{name}", repo: tap_remote_repo, state: "open", label: "\"long build\"")
+    long_prs = pull_requests.map { |pull_request| pull_request["number"] }
+    long_build_pr_files = {}
+    long_prs.each do |long_pr|
+      GitHub.get_pull_request_changed_files(name, tap_remote_repo, long_pr).each do |pr_file|
+        long_build_pr_files[pr_file["filename"]] = Array(long_build_pr_files[pr_file["filename"]]).push(long_pr)
+      end
+    end
+
+    this_pr_files = GitHub.get_pull_request_changed_files(name, tap_remote_repo, pr).map(&:filename)
+
+    conflicts = []
+    this_pr_files.each do |this_pr_file|
+      next unless long_build_pr_files.key?(this_pr_file)
+
+      long_build_pr_files[this_pr_file].each do |pr_number|
+        conflicts.append("#{tap.default_remote}/pull/#{pr_number}")
+      end
+    end
+    return if conflicts.blank?
+
+    # Raise an error, display the conflicting PR. For example:
+    # Error: You are trying to merge a pull request that conflicts with a long running build in
+    #  https://github.com/Homebrew/homebrew-core/pull/10867
+    #  https://github.com/Homebrew/homebrew-core/pull/9856
+    odie <<~EOS
+      You are trying to merge a pull request that conflicts with a long running build in
+       #{conflicts * "\n "}
+    EOS
```

Comment:
Should we show the name of the conflicting files?

Reply:
Agreed.

Reply:
Done